patches/chrome-installer-setup-setup_main.cc.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/chrome/installer/setup/setup_main.cc b/chrome/installer/setup/setup_main.cc
+index 0000c1392d4a75ceb856414b1243ba46737877f8..2968930ed4e709f328eac8e41748489a0c6c92b7 100644
+--- a/chrome/installer/setup/setup_main.cc
++++ b/chrome/installer/setup/setup_main.cc
+@@ -1729,6 +1729,8 @@ int SetupMain() {
+     return_code = InstallUtil::GetInstallReturnCode(install_status);
+   }
+ 
++  BRAVE_SAVE_PROMO_CODE
```

Comment:
do not add blank lines in patches, but I'm confused about the return code. Isn't this an msi installer? Does `GetInstallReturnCode` even get called?

Reply:
if `GetInstallReturnCode` does get called then we can do `#define GetInstallReturnCode(INSTALL_STATUS) GetInstallReturnCode(INSTALL_STATUS); SavePromoCode(return_code)`

Reply:
if `GetInstallReturnCode` does not get called we could do `#define InstallProducts(...) InstallProducts(__VA_ARGS__); SavePromoCode(GetInstallReturnCode(install_status));` 
There are various ways we could do this without patching

Reply:
> Isn't this an msi installer? Does GetInstallReturnCode even get called?

It should, it looks like it gets called whenever the installer action is not an "MSI uninstall"

Reply:
Not an MSI installer - it's a custom format

Reply:
> if GetInstallReturnCode does not get called we could do #define InstallProducts(...) InstallProducts(__VA_ARGS__); SavePromoCode(GetInstallReturnCode(install_status));

decided to go with this alternative, since there are multiple calls to `GetInstallReturnCode(install_status)`